### PR TITLE
Added init procedure to add student information

### PR DESCRIPTION
**Pull request by keke8273**

### CODE REVIEW

File: Pikeman/Pikeman.Algorithm/Student.cs [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-        private const int ClassNumberLength = 2;
-
         private readonly Guid _id;
-        private readonly string _schoolId;
+        private readonly StudentId _studentId;
 
         public Student(Guid id, string schoolId)
         {
-            _schoolId = schoolId;
+            _studentId = new StudentId(schoolId);
```
Comment (mylaluna) [inline Pikeman/Pikeman.Algorithm/Student.cs]: StudentID is a Class?

Comment (mylaluna) [inline Pikeman/Pikeman.Algorithm/Student.cs]: what is the purpose of  making studentid an object

**Comment by mylaluna** [comment]
Two things:
* What is purpose of making studentID an object
* it seems that you have a separate entry for importing student {ID, Name}. This is away from the design.

File: Pikeman/Pikeman.Algorithm/Student.cs (hunk 1 above)
Comment (keke8273) [inline Pikeman/Pikeman.Algorithm/Student.cs]: The studentId class wraps the logic that extract enrollment year and class number from the studentId string.

**Comment by keke8273** [comment]
As per discussion, we will keep the student initialization for now, (at least until we have the report demonstration.) Going forward we may have to import students during test results processing.

**Pull request merged**